src/libraries/System.Private.CoreLib/src/System/IO/Path.Unix.cs

```diff
@@ -103,7 +103,11 @@ public static unsafe string GetTempFileName()
             int tempPathByteCount = Encoding.UTF8.GetByteCount(tempPath);
             int totalByteCount = tempPathByteCount + fileTemplate.Length + 1;
 
+#if TARGET_BROWSER
+            Span<byte> path = new byte[totalByteCount];
```

Comment:
How does this fix the problem with insufficient randomness? Is it a workaround for a bug somewhere else?

Reply:
From the linked issue I gather that the underlying API apparently uses the destination pointer as part of the (non-CSPRNG) randomness formula, so passing the same destination twice (before the clock ticks over) makes for the same random path.

Reply:
https://github.com/emscripten-core/emscripten/blob/085fe968d43c7d3674376f29667d6e5f42b24966/system/lib/libc/musl/src/temp/__randname.c#L13

Reply:
File an issue in emsripten repo and link it from a comment?

Reply:
Issue filed, PR description updated.